### PR TITLE
Add marshal-unmarshal for ContainerContext

### PR DESCRIPTION
**Pull request by masterSplinter01**

### CODE REVIEW

File: pkg/session/container.go [hunk 1]
```diff
@@ -113,3 +113,19 @@ func (x *ContainerContext) ForSetEACL() {
 func (x *ContainerContext) IsForSetEACL() bool {
 	return x.isForVerb(session.ContainerVerbSetEACL)
 }
+
+func (x *ContainerContext) StableMarshal(buf []byte) ([]byte, error) {
```
Comment (cthulhu-rider) [inline pkg/session/container.go:117]:
```suggestion
func (x *ContainerContext) Marshal(bs ...[]byte) ([]byte, error) {
```

**Review by cthulhu-rider — CHANGES_REQUESTED** [review]

**Comment by cthulhu-rider** [comment]
Please add docs and unit test (see `Token`). Also add `[#390] pkg/session:` to commit msg.

File: pkg/session/container.go (hunk 1 above)
Comment (masterSplinter01) [reply pkg/session/container.go]: Fixed.

File: pkg/session/container.go [hunk 2]
```diff
@@ -113,3 +113,27 @@ func (x *ContainerContext) ForSetEACL() {
...
+// Marshal marshals ContainerContext into a protobuf binary form.
+func (x *ContainerContext) Marshal(bs ...[]byte) ([]byte, error) {
+	var buf []byte
+	if len(bs) > 0 {
+		buf = bs[0]
+	}
+	return x.ToV2().StableMarshal(buf)
+}
+
+// Unmarshal unmarshals protobuf binary representation of ContainerContext
```
Comment (cthulhu-rider) [inline pkg/session/container.go:126]:
```suggestion
// Unmarshal unmarshals protobuf binary representation of ContainerContext.
```

Comment (masterSplinter01) [reply pkg/session/container.go]: Sorry, typo. And linter said nothing about it.
Fixed.

**Review by alexvanin — APPROVED** [review]

**Review by cthulhu-rider — APPROVED** [review]

**Pull request merged**